packages/@icon-magic/cli/src/index.ts

```diff
@@ -40,7 +44,11 @@ program
   .description(
     'generates the flavors of the icon in the extension types that it can be consumed.'
   )
-  .action(async inputPaths => {
+  .option(
+    '-h, --hashing',
+    'If false, Icon Magic will run build and generate on all icons including those that have not changed since the previous execution. If true it will only run on icons that have changed assets. Defaults to true.'
```

Comment:
Make it less verbose. Also include only generates here

"When true(default), generates only those icon variants that have changed since the previous execution. If false, generates all icons. "